Sources/Mantis/CropView/CropView.swift

```diff
@@ -700,6 +717,19 @@ extension CropView {
     func crop() -> CropOutput {
         return crop(image)
     }
+    
+    /// completion is called in the main thread
+    func asynCrop(completion: @escaping (_ cropOutput: CropOutput) -> Void ) {
+        activityIndicator.isHidden = false
+        activityIndicator.startAnimating()
+        
+        DispatchQueue.global(qos: .userInteractive).async {
+            let cropOutput = self.crop()
+            DispatchQueue.main.async {
+                completion(cropOutput)
```

Comment:
Don't you need to hide the `activityIndicator`?

Reply:
I will fix it